### ISSUE TITLE
Only "Pending" status is available when "Answering" a ticket

### ISSUE THREAD

**Issue by dxdemetriou** [glpi-project/glpi#12055]
### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### Is there an existing issue for this?

- [X] I have searched the existing issues

### Version

10.0.2

### Bug description

The other status options except of the "Pending" status (as of GLPI 9.5) are missing on the Answer (Followup) button on GLPI 10:
- New
- Processing (assigned)
- Processing (Planned)
- Solved
- Closed

Tried also to change the status on the sidebar but on answering remains the previous status (and the two steps are 2 emails to the requester).

Or if it's possible to be able to the user to change the "Answer" button to have as predefined the "Solution" instead of the "Followup"

**Comment by SteadEXE** [comment]
I agree would be neat to have a status "answer" when a ticket has been answered to make everything easier.

**Comment by dxdemetriou** [comment]
> I agree would be neat to have a status "answer" when a ticket has been answered to make everything easier.

About the status "Answer", probably the Tags addon is preffered as the status numbers are standard and I saw a similar discussion in the past

I'm reffering to the button "Answer" that's actually the Follow-up we know from older GLPI versions. Just now the only available option is the "Pending" and everything else is missing, so everyone used older GLPI versions with "Follow-up & Solved", now cannot using the same behaviour without sending 2 emails to the user. The difference of not using the "Add a solution" and the  "Follow-up & Solved" is preffered it's because of the conversation history:

##FOREACH LAST 1 followups##
..
##ENDFOREACHfollowups##

**Comment by cconard96** [comment]
The old "split button" that let you select a new status when you add a followup or task was re-added via a simple plugin.
https://github.com/cconard96/glpi-stab-plugin

If you would like to see this added back into GLPI, the UserEcho page is the best place to ask and to get feedback from the community.
https://glpi.userecho.com/

**Comment by cedric-anne** [comment]
> The difference of not using the "Add a solution" and the "Follow-up & Solved" is preffered it's because of the conversation history:
> 
> ##FOREACH LAST 1 followups## .. ##ENDFOREACHfollowups##

In GLPI 10.0, you can use a loop on `timelineitems`, which will include all timeline elements, except the ticket description. See #9456 for more details.

**Comment by dxdemetriou** [comment]
> > The difference of not using the "Add a solution" and the "Follow-up & Solved" is preffered it's because of the conversation history:
> > ##FOREACH LAST 1 followups## .. ##ENDFOREACHfollowups##
> 
> In GLPI 10.0, you can use a loop on `timelineitems`, which will include all timeline elements, except the ticket description. See #9456 for more details.

When a user is replying back to the ticket, the timeline shouldn't be to include everything in a timeline?
The process I did:
-Send email to the receiver (ticket created - email contains the description)
-Answer by a followup (timeline is OK)
-User replies back to the ticket (only the description shown on timeline)
-Answer another followup (timeline is OK)
-User replies back (both of their replies are missing from timeline)
-Solved ticket (everything in timeline)
-User replies back (only description & solution are visible in timeline)

Actually I think that those missed informations could be a confuse for watchers if they are be notified by email
